manage.py

```diff
@@ -18,4 +18,8 @@
             "available on your PYTHONPATH environment variable? Did you "
             "forget to activate a virtual environment?"
         ) from exc
+
+    if os.environ.get("COVERAGE_PROCESS_START"):
+        os._exit = sys.exit
```

Comment:
Probably it can lead to some extra problems. I will recommend just creating a specific function that will flush the coverage and call os._exit. It will be safer.

Reply:
Could someone expand on why _exit() should normally only be used in the child process after a fork()?

Instead of calling exit(), the child can call _exit(), so that it doesn’t flush stdio buffers. This technique exemplifies a more general principle: in an application that creates child processes, typically only one of the processes (most often the parent) should terminate via exit(), while the other processes should terminate via _exit(). This ensures that only one process calls exit handlers and flushes stdio buffers, which is usually desirable

Reply:
Returned os._exit and added saving coverage before calling it